src/core/stac/qgsstacasset.cpp

```diff
@@ -52,3 +52,25 @@ QStringList QgsStacAsset::roles() const
 {
   return mRoles;
 }
+
+bool QgsStacAsset::isCloudOptimized() const
+{
+  if ( mMediaType == QLatin1String( "image/tiff; application=geotiff; profile=cloud-optimized" ) ||
+       mMediaType == QLatin1String( "image/vnd.stac.geotiff; cloud-optimized=true" ) ||
+       mMediaType == QLatin1String( "application/vnd.laszip+copc" ) ||
+       mHref.endsWith( QLatin1String( "/ept.json" ) ) )
```

Comment:
If you wanted to avoid the duplicate strings here + and in formatName, you could potentially replace this with:

```suggestion
  const QString format = formatName();
  return format == QLatin1String( "COG" ) ||
       format == QLatin1String( "COPC" ) ||
       format == QLatin1String( "EPT" );
```

Reply:
Ah, right, `formatName()` was a late addition and forgot to use it elsewhere too!